pkg/network/node/ovs/ovs.go

```diff
@@ -109,6 +109,12 @@ const (
 	RETRIES   = 10
 )
 
+var OVSBackoff utilwait.Backoff = utilwait.Backoff{
+	Duration: 500 * time.Millisecond,
+	Factor:   1.25,
+	Steps:    RETRIES,
```

Comment:
No need to have a separate `RETRIES` any more

Reply:
Actually `RETRIES` is used here to make sure the tests passes: https://github.com/openshift/sdn/blob/master/pkg/network/node/ovs/ovs_test.go#L112

Reply:
Well, I am not sure if that's what you meant by "separate `RETRIES`". Could you clarify?

Reply:
ah... well, the test could just use `OVSBackoff.Steps` intead of `RETRIES`.
actually, `OVSBackoff` should be `ovsBackoff`; we don't want to export that variable

Reply:
I've addressed it, @danwinship . PTAL